view/code/index.ts

```diff
@@ -104,7 +119,7 @@ function listenChanges(input: HTMLInputElement): void {
     if (input === lchInput) {
       setLch()
     } else {
-      setRgb()
+      processRgbBlur()
```

Comment:
It will be better to just move `processRgbBlur` content here, since you do not use it anywhere esle and function name doesn’t explain what function does

Reply:
Working on your remarks noticed that we can insert color in both inputs. Thus placed the snippet higher